components/kyma-environment-broker/internal/cls/kyma_version_check_test.go

```diff
@@ -0,0 +1,26 @@
+package cls
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheckKymaVersionPR(t *testing.T) {
+	var tests = []struct {
+		summary string
+		version string
+		res     bool
+	}{
+		{"Check PR image", "PR-1234", true},
+		{"Check Master image", "master-abcd2", true},
+		{"Check Kyma image 1.21.1", "1.21.1", true},
+		{"Check Kyma image 1.19.1", "1.19.1", false},
```

Comment:
I don't like returning errors when a version does not fit the pattern (PR-xxx of master-xxx or semver).
we can assume every non-semantic version is a 1.20 or above

Reply:
fixed.